src/components/Typed.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import { init } from "ityped";
+
+export default class Typed extends Component {
+  componentDidMount() {
+    init(this.text, {
+      strings: ["animators", "storytellers", "motion designers"],
```

Comment:
If this array of strings were a prop the `Typed` component receives this could be re-usable. Not sure if we have any other use-cases for it but you never know.